src/components/Calculator.css

```diff
@@ -0,0 +1,36 @@
+* {
+  color: rgb(31, 18, 18);
+  margin: 0;
+  padding: 0;
+}
+
+section {
+  display: grid;
+  grid-template-columns: 100px 100px 100px 100px;
+  border: 2px solid rgb(216, 203, 203);
+  width: fit-content;
+}
+
+section div {
+  border: 1px solid rgb(216, 203, 203);
+  padding: 20px;
+  font-size: 30px;
+  text-align: center;
+}
+
+.screen {
+  grid-column-start: 1;
+  grid-column-end: 5;
+  text-align: right;
+  color: white;
+  background-color: rgb(78, 82, 93);
+}
+
+.zero {
+  grid-column-start: 1;
+  grid-column-end: 3;
+}
+
+.function {
+  background-color: rgb(225, 115, 56);
```

Comment:
- [ ] Your component should match the following given design: please make sure that it matches the design.
- [ ] Please use the same color on the screen, keyword.
### Your design
![his](https://user-images.githubusercontent.com/81727988/219429748-a4fbe28c-94fb-4e27-b9e2-8d6108d0b968.png)

### Original design
![original](https://user-images.githubusercontent.com/81727988/219429773-a24add9b-8b77-4d4d-a15d-467134d327a6.png)